tests/foreman/ui/test_host.py

```diff
@@ -2151,18 +2175,14 @@ def test_positive_read_details_page_from_new_ui(
     session, module_host_template, enable_new_host_details_ui, setting_update
 ):
     """Create new Host and read all its content through details page
-
     :id: ef0c5942-9049-11ec-8029-98fa9b6ecd5a
-
     :expectedresults: Host is created and has expected content
-
```

Comment:
Those missing lines in docstring are probably causing checks to fail, same for other tests.